src/grandorgue/sound/GOSoundProviderWave.cpp

```diff
@@ -278,7 +278,7 @@ unsigned GOSoundProviderWave::GetFaderLength(unsigned MidiKeyNumber) {
   return fade_length;
 }
 
-void GOSoundProviderWave::LoadFromFile(
+void GOSoundProviderWave::LoadFromSampleFiles(
```

Comment:
I'd instead suggest: LoadSamplesFromFile(

Reason being, the plural you add indicates that multiple files are handled here.

Reply:
This methods really may load samples from multiple files: attacks, releases. So the plural is really necessary here.

Reply:
I changed the names to `LoadFromMultipleFiles` and `LoadFromOneFile`

Reply:
Ok, but still it's Samples in plural! So

LoadSamplesFromFiles

would be better as a description.